scripts/test

```diff
@@ -37,7 +43,12 @@ copy_logs(){
 run_tests()
 {
     pushd ./tests/validation
-    tox || [ `uname` == "Linux" ] && copy_logs && exit 1
+    if [ "${CATTLE_TEST_PARALLEL_THREADS}" -gt 1 ] 
```

Comment:
But, I agree that the normal tox output is the best. So, im ok with the if/else, but please do add the -v to the multi-thread version.